indexing_options.go

```diff
@@ -24,6 +24,22 @@ const (
 	SkipFreqNorm
 )
 
+const (
+	BM25Scoring  = "bm25"
+	TFIDFScoring = "tfidf"
+)
+
+// Scoring model indicates the algorithm used to rank documents fetched
+// for a query performed on a field. Default is the bm25 scoring but can be set
+// to other models like the legacy tf-idf.
```

Comment:
Comment needs updating - could you also mention that this applicable to text-based/non-vector fields only.